qonsole_rails.gemspec

```diff
@@ -18,18 +18,17 @@ Gem::Specification.new do |s|
   s.files = Dir['{app,config,db,lib}/**/*', 'LICENSE', 'Rakefile', 'README.rdoc']
   s.test_files = Dir['test/**/*']
 
-  s.add_dependency 'rails', '~> 5.2.4'
+  s.add_dependency 'rails'
 
-  s.add_dependency 'codemirror-rails', '~> 5.11'
-  s.add_dependency 'faraday', '~> 0.17.0'
-  s.add_dependency 'faraday-encoding', '~> 0.0.5'
-  s.add_dependency 'faraday_middleware', '~> 0.13.1'
-  s.add_dependency 'font-awesome-rails', '~> 4.7.0.1'
-  s.add_dependency 'haml-rails', '~> 2.0.0'
-  s.add_dependency 'jquery-datatables-rails', '~> 3.4.0'
-  s.add_dependency 'jquery-rails', '~> 4.3.1'
-  s.add_dependency 'lodash-rails', '~> 4.17.4'
-  s.add_dependency 'modulejs-rails', '~> 2.2.0.0'
+  s.add_dependency 'faraday'
+  s.add_dependency 'faraday-encoding'
+  s.add_dependency 'faraday_middleware'
+  s.add_dependency 'font-awesome-rails'
+  s.add_dependency 'haml-rails'
+  s.add_dependency 'jquery-datatables-rails'
+  s.add_dependency 'jquery-rails'
+  s.add_dependency 'lodash-rails'
+  s.add_dependency 'modulejs-rails'
```

Comment:
Does Rubocop not complain about not having a targeted version? 🤔

Reply:
It doesn't for me, and it seems to be a lot of disagreement on this one as well (https://www.reddit.com/r/rails/comments/sujdqk/best_practice_should_i_specify_versions_in_gem/). Personally I prefer to leave the versions out because gems have great backwards compatibility (most of the time) and `Gemfile.lock` has all the versions anyway. If the app is a bit older and using old ruby / rails versions it makes sense to have them fixed for compatibility issues. If the app is using newer ruby / rails versions I think it makes future upgrades easier as you won't have to worry about upgrading gems manually from the gemspec, instead you would just nuke `Gemfile.lock` and regenerate the file. Also helps with fixing dependabot alerts